docs/src/documentation/03-components/02-structure/tile/03-accessibility.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/tile/accessibility/
+---
+
+# Accessibility
+
+## Tile
+
+The Tile component has been designed with accessibility in mind.
+
+The component offers flexibility in terms of the HTML element used for the root node. By default, it will render a `div` element.
+
+### Expandable / Clickable
+
+The prop `expandable` makes the Tile clickable. In that case, assistive props like `ariaExpanded` are handled automatically.
```

Comment:
I would maybe use `aria-expanded` here as that's the name and it's not a custom one, wdyt?